### PR TITLE
EOS-21437: HARE README GitHub Issue: #1664 - cortx-motr-tests rpm installation gives error

### PR DESCRIPTION
**Pull request by vaibhavparatwar**
Other than cortx-motr-test rpms installing other rpms without removing motr-test rpm
Signed-off-by: Vaibhav Paratwar <vaibhav.paratwar@seagate.com>

-----
[View rendered README.md](https://github.com/Seagate/cortx-hare/blob/vaibhavparatwar-patch-3/README.md)

### CODE REVIEW

**Comment by vaibhavparatwar** [comment]
Addressing: https://github.com/Seagate/cortx-hare/issues/1664

File: README.md [hunk 1]
```diff
@@ -94,12 +94,15 @@ and health-checking mechanisms.
...
   * From sources
+    **NOTE:** Installing motr RPMs other than "cortx-motr-test*.rpm"
     ```sh
     git clone --recursive https://github.com/Seagate/cortx-motr.git
     cd cortx-motr
 
     scripts/m0 make rpms
-    sudo rpm -ivh ~/rpmbuild/RPMS/x86_64/cortx-motr-*.rpm
+    cd ~/rpmbuild/RPMS/x86_64/
+    sudo rpm -ivh $(ls | grep  "cortx-motr-" | grep -v "cortx-motr-tests")
```
Comment (suvratjoshi) [inline README.md:104]: Before this we used to  install both motr and motr-test rpms?
Or motr did not use to create the test rpms, and now it has started creating this additional test rpm, so the installation is failing?
if we dont install the motr-test rpm's, then if someone wants to run motr-tests which are probably provided by this motr-test rpm, then what will happen? will it still run or fail?
how motr is is doing it, in their installations?

**Comment by suvratjoshi** [comment]
PR title :
It would be nice if JIRA and PR has title which relates to actual issue.

PR description and commit message : 
I think "Signed off" should be below the PR description.

File: README.md (hunk 1 above)
Comment (vaibhavparatwar) [reply README.md:104]: @suvratjoshi I think this is not required when testing HARE. This README's scope is hare. This has been confirmed with @atulsdeshmukh2312 . I think installation of motr-test rpm is in the scope of Motr README. For your questions on whether its gettign generated recently - @madhavemuri  can help here.

**Review by vaibhavparatwar — COMMENTED** [review]
> PR title :
> It would be nice if JIRA and PR has title which relates to actual issue.
> 
> PR description and commit message :
> I think "Signed off" should be below the PR description.

Done.

**Pull request merged**